src/transformers/models/clip/feature_extraction_clip.py

```diff
@@ -156,6 +162,20 @@ def __call__(
 
         return encoded_inputs
 
+    def convert_rgb(self, image):
+        """
+        Converts `image` to RGB format. Note that this will trigger a conversion of `image` to a PIL Image.
+
+        Args:
+            image (`PIL.Image.Image` or `np.ndarray` or `torch.Tensor`):
+                The image to convert.
+        """
+        self._ensure_format_supported(image)
+        if not isinstance(image, Image.Image):
+            image = self.to_pil_image(image)
+
+        return image.convert("RGB")
```

Comment:
I don't think we should force a conversion for non-PIL.Image types. If the user has passed a NumPy array or a torch Tensor, they will have converted it to the format of their choice already.

Reply:
Yup, basically changing
```
        if not isinstance(image, Image.Image):
            image = self.to_pil_image(image)
```
to
```
        if not isinstance(image, Image.Image):
            return image
```